CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to hoggorm
+
+If you are interested in contributing to hoggorm, your contributions will fall into two categories:
+
+1. You want to implement a new feature:
+....
+2. You want to fix a bug:
```

Comment:
We should introduce a third category: Improvement of documentation.

Reply:
Done